README.md

```diff
@@ -1,116 +1,67 @@
-filesystem-tree. The assembler is similarly isolated, and given the same
-tree, in read-only mode and assembles it into an image without altering
-its contents.
+Build-Pipelines for Operating System Artifacts
 
-The filesystem tree produced by the final stage of a pipeline, is named
-and optionally saved to be reused as the base for future pipelines.
+OSBuild is a pipelined build system for operating system artifacts. It defines
+a universal pipeline description and a build system to execute them, producing
+artifacts like operating system images, working towards an image build pipeline
+that is more comprehensible, reproducible, and extendable.
 
-Each stage is passed the (appended) `options` object as JSON over stdin.
+See the `osbuild(1)` man-page for details on how to run osbuild, the definition
+of the pipeline description, and more.
 
-The above pipeline has no base and produces a qcow2 image.
+### Project
 
-## Running
+ * **Website**: <https://www.osbuild.org>
+ * **Bug Tracker**: <https://github.com/osbuild/osbuild/issues>
 
-```
-usage: __main__.py [-h] [--build-env ENV] [--store DIRECTORY] [-l DIRECTORY]
-                   [--json]
-                   PIPELINE
-
-Build operating system images
-
-positional arguments:
-  PIPELINE              json file containing the pipeline that should be
-                        built, or a '-' to read from stdin
-
-optional arguments:
-  -h, --help            show this help message and exit
-  --build-env ENV       json file containing a description of the build
-                        environment
-  --store DIRECTORY     the directory where intermediary os trees are stored
-  --sources SOURCES     json file containing a dictionary of source
-                        configuration
-  -l DIRECTORY, --libdir DIRECTORY
-                        the directory containing stages, assemblers, and the
-                        osbuild library
-  --json                output results in JSON format
-```
+### Requirements
+
+The requirements for this project are:
+
+ * `python >= 3.8`
+ * `systemd-nspawn >= 244`
+
+Additionally, the built-in stages require:
 
-### Running example
+ * `bash >= 5.0`
+ * `coreutils >= 8.31`
+ * `curl >= 7.68`
+ * `qemu-img >= 4.2.0`
+ * `rpm >= 4.15`
+ * `tar >= 1.32`
+ * `util-linux >= 235`
 
-You can build basic qcow2 image of Fedora 30 by running a following command:
+At build-time, the following software is required:
 
+ * `python-docutils >= 0.13`
+ * `pkg-config >= 0.29`
+
+### Build
+
+The standard python package system is used. Contact upstream documentation for
+detailed help. In most situations the following commands are sufficient to
+build and install from source:
+
+```sh
+python setup.py build
+python setup.py install --skip-build --root=/
 ```
-sudo python3 -m osbuild --libdir . --sources samples/sources.json samples/base-qcow2.json
+
+The man-pages require `python-docutils` and can be built via:
```

Comment:
It is `python3-docutils` on Fedora ;P

Reply:
I know. Upstream calls it `python-docutils`, other distros as well. I am unsure what to do. I decided to mention project-names here, rather than package names. Any strong opinion on this?